lib/api_helpers.rb

```diff
@@ -45,11 +45,11 @@ def retry_interval_for(count)
   #
   # * *returns*
   #   - (Hash) => Hash of default HTTP headers, e.g. Authorization, Content-Type
-  def get_default_headers
+  def get_default_headers(content_override: nil)
```

Comment:
"_override" is code-smelly.  More conventional parameter name style, and more IDE-discoverable:

```suggestion
  def get_default_headers(content_type: 'application/json')
```

Reply:
[5cc7e8d](https://github.com/broadinstitute/single_cell_portal_core/pull/1679/commits/5cc7e8dbacf6b82215cdabf9dc701c27ade0a6d4)